Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Boxes/shotgun.yml

```diff
@@ -47,7 +47,7 @@
       - state: beanbag
 
 - type: entity
-  name: shell box (lethal)
+  name: ammunition box (.50 pellet)
```

Comment:
in `BaseMagazineShotgun` this is called (.50 shells)
Not sure which one is technically correct, but it should be consistent